### ISSUE TITLE
zammad barrier-free? (WCAG 2.0 AA)

### ISSUE THREAD

**Issue by rolfschmidt** [zammad/zammad#3067]
https://www.w3.org/Translations/WCAG20-de/

- Is zammad barrier-free?
- Does it match WCAG 2.0 AA?

### Infos:

* Used Zammad version: 3.3
* Installation method (source, package, ..):all
* Operating system: all
* Database + version: all
* Elasticsearch version: all
* Browser + version: all

### Expected behavior:

https://www.w3.org/Translations/WCAG20-de/

### Actual behavior:

???

**Comment by schmitzd** [comment]
Two problematic examples:
* The search box on the top left, see code https://github.com/zammad/zammad/blob/72bb320fb0bb4cdce57a80383e4c5da90b5ee04b/app/assets/javascripts/app/views/navigation.jst.eco#L3 needs a label, at least an ARIA label.
* At the dashboard you cannot navigate between "my stats" and "first steps" without a mouse (not reachable via keyboard only and no shortcuts); the same holds true when creating a ticket and trying to switch between "incoming call", "outgoing call" or "e-mail".

**Comment by rolfschmidt** [comment]
@mrflix Do you know what we have done in the initial implementation phase of zammad? Is zammad in general usable on the screen reader or matching standards of WCAG? (question)

@schmitzd Yeah looks like in the community there is also a discussion about some points:

https://community.zammad.org/t/default-color-grey-in-grey/1485/3

**Comment by schmitzd** [comment]
@rolfschmidt Thanks for the link to the contrast discussion. I have seen that before but since these kinds of problem *could* mostly be solved via CSS (and thus by ourselves) I think we need to focus the ones that require changes in the generated HTML code. But you are right some of the issues mentioned there belong indeed to the kind or problems that need to be solved in the code base.

**Comment by mrflix** [comment]
@rolfschmidt in the initial implementation there was no focus on screen reader or matching standards of WCAG support.

**Comment by Michi01** [comment]
Any update on this issue? It is a relatively important topic nowadays, especially public institutions have corresponding legal requirements to pay attention to accessibility points. This would make the use of Zammad much easier.

**Comment by MrGeneration** [comment]
We in general do not communicate any ETAs.
I'm afraid bumping this issue will not help at all in moving this topic sorry.

**Comment by Hemmingen** [comment]
We use Zammad in public schools in Germany and texts like "auf intern setzen", "antworten", "weiterleiten", "abspalten" with a 1,97:1 contrast are horrible to read for a lot of people. We're hopeful "mit dem Release 7.0 wird Zammad im neuen Glanz erscheinen." (https://zammad.com/de/blog/magie-des-teamgeists-ein-blick-auf-das-zammad-team-offsite-2023) means we'll get a new design in 2024 with accessibility in mind where the contrast ratio between all texts and their background is at least 4,5:1 in contrast. Furthermore we would like to see the implementation of a simpler way to use alternate fonts in Zammad like Atkinson Hyperlegible. Thank you.

**Comment by MrGeneration** [comment]
> Furthermore we would like to see the implementation of a simpler way to use alternate fonts in Zammad like Atkinson Hyperlegible.

That's a feature request, please post it in the community - thank you!
https://community.zammad.org/c/stuff-you-like-zammad-to-have-feel-free-to-discuss-and-add-proposals/6/none
